src/super_gradients/training/utils/sg_trainer_utils.py

```diff
@@ -358,3 +359,18 @@ def get_callable_param_names(obj: callable) -> Tuple[str]:
     :return: Param names of that object
     """
     return tuple(inspect.signature(obj).parameters)
+
+
+def log_main_training_params(gpu_mode: MultiGPUMode, num_gpus: int, batch_size: int, batch_accumulate: int, len_train_set: int):
+    """Log training parameters"""
+    msg = "TRAINING PARAMETERS:\n" \
+          f"    - Mode:                         {gpu_mode.name if gpu_mode else None}\n"\
+          f"    - Number of GPUs:               {num_gpus}/{torch.cuda.device_count():<8} [num_gpus]\n" \
```

Comment:
instead of "n/m" please write "n    (m available on the machine)"

Reply:
Changed in 4c1c3cdbbb19a730b97c2a86af144be942891de0